src/components/HeaderBar/HeaderBar.tsx

```diff
@@ -1,19 +1,24 @@
-import { Navbar, Container, ButtonGroup, Button } from 'react-bootstrap';
+import { Navbar, Container, ButtonGroup, Button, useAccordionButton } from 'react-bootstrap';
 import { Link, useHistory } from "react-router-dom";
 import { useWalletHook } from '../NavBar/useWallethook';
+import { Hamburger } from '../HumburgerMenu/HumburgerMenu';
+import { useState } from 'react';
 
 export const HeaderBar: React.FC = () => {
     const history = useHistory();
     const wallet = useWalletHook();
+    const HumburgerMenu = useState();
```

Comment:
You need to add variable to control the state of Hamburger
```suggestion
    const [isOpen, setIsOpen] = useState<boolean>(false);
```

Reply:
![Uploading Screen Shot 2022-09-08 at 11.50.10 am.png…]()